experimenter/experimenter/nimbus-ui/src/components/Summary/index.test.tsx

```diff
@@ -74,6 +75,20 @@ describe("Summary", () => {
       screen.queryByTestId("summary-page-signoff-not-launched"),
     ).toBeInTheDocument();
   });
+  it("updates signoff checklist state correctly", () => {
+    render(<Subject />);
+    const legalSignoffCheckbox = screen.getByTestId("is-legalsignoff-checkbox");
+    const qaSignoffCheckbox = screen.getByTestId("is-qasignoff-checkbox");
+    const vpSignoffCheckbox = screen.getByTestId("is-vpsignoff-checkbox");
+
+    userEvent.click(legalSignoffCheckbox);
+    userEvent.click(qaSignoffCheckbox);
+    userEvent.click(vpSignoffCheckbox);
```

Comment:
These need to be awaited.

Reply:
I am not sure why these need to be awaited 🤔

Reply:
Because it returns a promise, which must be awaited so any asynchronous side effects can complete.

https://testing-library.com/docs/user-event/convenience#click